### DISCUSSION TITLE
Kata 0

### DISCUSSION THREAD

**Discussion by Laura-Karina-GR** [LaunchX-InnovaccionVirtual/CursoIntroPython#174] [Q&A]
Hola, no puedo crear mi biblioteca en la kata :( 
¿Alguien me puede orientar por favor? la tengo así 
![2022-02-11 (2)](https://user-images.githubusercontent.com/99380455/153696154-cefe5e1f-a793-4ab8-abe9-04bd3f25eff0.png)

**Comment by OscarPedroza** [comment]
Tienes tu comando en un modulo de Markdown
Añade un bloque de código python y ejecuta ahí el comando:
![Captura](https://user-images.githubusercontent.com/83786890/153696597-e5f55537-4da5-4149-87e6-726a53bd37be.PNG)

**Reply by Laura-Karina-GR** [reply]
Muchas gracias
